_layouts/report.md

```diff
@@ -147,5 +155,21 @@
             </div>
             <br><br>
         </div>
+        <script>
+        $('document').ready(function(){
+            var hash_value = document.getElementById('sha256sum_hash_value').textContent.trim();
+            document.getElementById('sha256sum_hash_value').textContent = hash_value.substring(0,7);
+            document.querySelector(".fa-clipboard").addEventListener("click", function(){
+                var new_elment = document.createElement('textarea');
+                new_elment.value = hash_value;
+                new_elment.setAttribute('readonly', '');
+                new_elment.style = {position: 'absolute', left: '-9999px'};
```

Comment:
style should be in CSS

Reply:
The element is created and then deleted, it is just used to copy the hash value once it's get copied the element is deleted from the body.